### PR TITLE
Update README.md for STM32F4 and STM32F1 boards in bugfix 2.0

### PR DESCRIPTION
**Pull request by chepo92**
### Description

STEVAL-3DP001V1 is a STM32F4 so added that category, and added GMT32 in STM32F1

### CODE REVIEW

**Comment by thisiskeithb** [comment]
@thinkyhead: Should the `dev-2.1.x` branch still exist? It seems to cause confusion with users thinking they are using the latest & greatest version of Marlin when it's just really an outdated copy of `bugfix-2.0.x`.

**Comment by chepo92** [comment]
I had that doubt :D 
As is stated in 2.0 branch  readme.md
- Submit New Features to the (dev-2.1.x) branch.
Instructions unclear captain, abandon ship? 

Also I consider the readme of dev2.1 has more details (HAL), that 2.0 doesn't

**Comment by thisiskeithb** [comment]
Yes, but new feature PRs are still being submitted to `bugfix-2.0.x` and not `dev-2.1.x`.

**Comment by chepo92** [comment]
Then let's make that explicit in readme

**Comment by thisiskeithb** [comment]
That's why I asked Scott 🙂

**Comment by chepo92** [comment]
The same change of this PR should be done to 2.0 bugfix anyway, rebasing...

File: README.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ Marlin 2.0 introduces a layer of abstraction so that all the existing high-level
   boards|processor|speed|flash|sram|logic|fpu
   ----|---------|-----|-----|----|-----|---
   [Arduino STM32](https://github.com/rogerclarkmelbourne/Arduino_STM32)|[STM32F1](https://www.st.com/en/microcontrollers-microprocessors/stm32f103.html) ARM-Cortex M3|72MHz|256-512k|48-64k|3.3V|no
+  [Geetech GTM32](https://github.com/Geeetech3D/Diagram/blob/master/Rostock301/Hardware_GTM32_PRO_VB.pdf)|[STM32F1](https://www.st.com/en/microcontrollers-microprocessors/stm32f103.html) ARM-Cortex M3|72MHz|256-512k|48-64k|3.3V|no
```
Comment (thinkyhead) [inline README.md]: Please fix the spelling of "Geeetech." I am not able to edit.

Comment (chepo92) [reply README.md]: Done

**Comment by thinkyhead** [comment]
There are probably many more boards that could be listed.

**Pull request merged**